File: Python/leetcode_1207.py
from typing import List
from collections import Counter


def unique_occurrences(arr: List[int]) -> bool:
    c = Counter(arr)
    return len(set(c.values())) == len(c.values())


print(unique_occurrences([1, 2, 2, 1, 1, 3]))

File: JavaScript/leetcode_1207.js
/**
 * @param {number[]} arr
 * @return {boolean}
 */
var uniqueOccurrences = function (arr) {
    let c = {};
    arr.forEach((ele) => {
        c[ele] = (c[ele] || 0) + 1;
    });
    return new Set(Object.values(c)).size == Object.values(c).length;
};
